cli/dcoscli/marathon/main.py

```diff
@@ -596,15 +599,61 @@ def _update(app_id, properties, force):
     client = marathon.create_client()
 
     # Ensure that the application exists
-    client.get_app(app_id)
+    current_app = client.get_app(app_id)
+
+    schema = _app_schema()
+    app_resource = _parse_properties(properties, schema)
+    _validate_update(current_app, app_resource, schema)
 
-    app_resource = _parse_properties(properties, _app_schema())
     deployment = client.update_app(app_id, app_resource, force)
 
     emitter.publish('Created deployment {}'.format(deployment))
     return 0
 
 
+def _validate_update(current_resource, properties, schema):
+    """
+    Validate resource ("app" or "group") update
```

Comment:
Fix the the parameter descriptions below. The syntax doesn't look correct. E.g. `:param current_resource: ...` and `:type current_resource: ...`